### PR TITLE
Avoid short titles in bootstrap

### PR DESCRIPTION
**Pull request by g4s8**
#754 - check title length in bootstrap

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#976](https://github.com/zerocracy/farm/issues/976) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#976](https://github.com/zerocracy/farm/issues/976) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1180640064); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/main/resources/com/zerocracy/stk/pm/bootstrap.groovy [hunk 1]
```diff
@@ -78,10 +78,13 @@ def exec(Project project, XML xml) {
     .param('reason', new Par('Project %s was bootstrapped').say(project.pid()))
     .postTo(project)
   if (claim.hasParam('channel')) {
-    claim.copy()
-      .type('Set title')
-      .param('title', claim.param('channel'))
-      .postTo(project)
+    String chan = claim.param('channel')
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/bootstrap.groovy]: @g4s8 let's call this `chnl`.

File: src/test/resources/com/zerocracy/bundles/bootstraps_short_named_project/_after.groovy [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.zerocracy.Project
+import com.zerocracy.pmo.Catalog
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  Catalog catalog = new Catalog(farm).bootstrap()
+  MatcherAssert.assertThat(
+    'Project title should be set from channel prop',
```
Comment (carlosmiranda) [inline src/test/resources/com/zerocracy/bundles/bootstraps_short_named_project/_after.groovy]: @g4s8 error message is confusing here, it seems to imply that title should be set to the name of the channel. Let's change it something like:

> Project title should be set to project ID if channel property is of length >2

**Comment by g4s8** [comment]
@carlosmiranda fixed in bf44b13

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 Rultor failed to push: `error: failed to push some refs to 'git@github.com:zerocracy/farm.git'`, please restart merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#976](https://github.com/zerocracy/farm/issues/976) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**